lgc/builder/BuilderImpl.cpp

```diff
@@ -688,109 +741,95 @@ Instruction *BuilderImpl::createWaterfallLoop(Instruction *nonUniformInst, Array
   SmallVector<Value *, 2> nonUniformIndices;
   // Maps the nonUniformIndex that is returned by traceNonUniformIndex() to the nonUniformInst.
   DenseMap<Value *, std::pair<Value *, unsigned>> nonUniformIndexImageCallOperand;
```

Comment:
Is `nonUniformIndexImageCallOperand` map being actually used?

Reply:
Nope. Good catch :)